Removes roundstart singularity, adds roundstart supermatter, supermatter now collapes into singularity

Tokiko1 opened the pull request:
🆑 Kor, Jordie and Tokiko1
add: Singularity containment has been replaced on box, meta, and delta with a supermatter room. The supermatter gives ample warning when melting down, so hopefully we'll see fewer 15 minute rounds ended by a loose singularity.
add: Supermatter crystals now collapse into singularities when they fail, rather than explode.
/🆑

As requested by @KorPhaeron 

Adds an actual supermatter crystal instead of the varedited shards that were used before.
Adds the supermatter to box.
Adds the supermatter to meta.
Removes singulo from delta and gives engineers access to some needed areas.
Adds orange pipes and manifolds for mappers.
Makes the supermatter spawn a singulo.
Rebalanced the power generation of radiation collectors by making them half as efficient as before.

Closes #24394 because it's the same thing, just a different design and some fixes.
The filesize seems a bit large for mapmerge, did I do it incorrectly?

Pictures:
Box:
![supermatter1](https://cloud.githubusercontent.com/assets/7812993/23335180/e7141f10-fbaf-11e6-97c3-b1d19b3da586.PNG)
![supermatter2](https://cloud.githubusercontent.com/assets/7812993/23335184/ebb35f22-fbaf-11e6-9441-241b77836b42.PNG)

Meta:
![metaatmos1](https://cloud.githubusercontent.com/assets/7812993/23336372/9c915a60-fbce-11e6-839a-a0c1e13b658f.PNG)

Delta:
![deltaatmos](https://cloud.githubusercontent.com/assets/7812993/23337119/ed6918a0-fbe3-11e6-9029-d0293cb10df6.PNG)

Tokiko1 committed:
Adds supermatter to box.
Adds orange pipes.
Makes the supermatter spawn a singulo.

Shapsy commented:
You're a hero for this

Armhulen commented:
delta has two supermatters?

Tokiko1 commented:
I honestly don't know what to do with delta. It already has a supermatter engine, though engineers can't quite access it. Thoughts?

ExcessiveUseOfCobblestone commented:
Remove Delta obviously :^)

KorPhaeron commented:
Change the doors so they can access it, delete the PA, board up the PA area and singulo room with wood barricades?

Armhulen commented:
I actually like Kor's idea.

Metallusions commented:
Switch it out for a PA setup with no singulo/tesla generator.

calzilla1 commented:
Add a chute for easy assistant disposal

Core0verload commented:
The reason why singulo escapes all the time for no reason: the sparks feed it. The very same sparks that are always generated by Tesla part bumping into containment field. Remove sparks energy, or remove tesla parts, and enjoy your stable rounds once again.

Haevacht commented:
Coreoverload, #23502 that is no longer the case. Also WRENCH DOWN THE USELESS PARTS.

Core0verload commented:
Then I don't see any reason to remove singulo at all.

If singulo escaping causes too much problems because singulo is basically unkillable, add a single shot rocket launcher with a special charge designed to make singulo/Tesla collapse in a spectacular fashion. Make it also heavily damage other extradimensional shit like Ratvar's arc or Nar-Sie. Place it in the vault to make sabotaging it possible.

AdamElTablawy commented:
Here's a reason to remove singulo-

Supermatter is cool and fun to customize and takes upkeep as compared to set-n-forget-ularity

Magikarpet commented:
you can ask cheridan but apparently the supermatter doesn't actually gain damage from tempratures

if you also wanted to make the engine more modifiable replacing the r-walls with glass could work

Tokiko1 committed:
Added orange pipe manifolds.
Added a supermatter crystal and replaced the varedited shards on box and meta.
Added supermatter engine to metastation.

Tokiko1 commented:
Metastation is done, only Delta(and testing) remains now.

Cheridan commented:
The box atmos setup makes it nearly impossible to run the SM mix and do other atmos stuff at the same time, such as filling canisters.

Tokiko1 committed:
Removed singulo from Deltastation.
Gave engineers access to some Deltastation atmos areas, including the engine.
Fixed some access bugs.

Tokiko1 committed:
Misc fixes.

Yoshmaster commented:
this looks really cool!

Yoshmaster commented:
Later, you should add a machine that converts plasma gas into plasma bars. Then engineering could optimize the shard and give the bars to cargo.

Iamgoofball commented on code/modules/power/supermatter/supermatter.dm:
```diff
@@ -91,9 +91,12 @@
+	var/turf/T = get_turf(src)
 	qdel(src)
+	if(T)
+		var/obj/singularity/S = new(T)
```
:-1:

make this randomize between singulo and tesla

SgtxRootbeer replied:
@Iamgoofball make it randomize between singulo, /vg/ style delam, tesla, and bomb-cap-ignoring 10,30,30 bomb

CPTANT replied:
Yeah I think randomizing the result would give some much needed variety to engine failures.

Tokiko1 replied:
I think this will have to wait until a later PR, unless @KorPhaeron wants to add something to this PR?

SgtxRootbeer commented:
@Yoshmaster
@kevinz000 
that actually looks like an acceptable replacement for the power-transfer laser.

Steelpoint commented:
This should only be viable to be merged if, and only if, the Supermatter set up guide on the wiki is fully expanded and brought to the same quality standard as the Sing Guide.

CPTANT commented:
What is actually the criteria for engine failure? The suppermatter seems surprisingly stable.

Core0verload commented:
inb4 traitor AI fails to release plasma because engineering sold it. All of it.

Tokiko1 commented:
One quick adjustment incoming, don't merge just yet.

Tokiko1 committed:
Fixed the roundstart reflectors being impossible to deconstruct.
Bunch chamber tweaks on meta.

Tokiko1 committed:
Poly now squawks about the supermatter engine instead of the singulo.
Radiation collectors are now half as efficient.

Tokiko1 commented:
I've been testing this setup and I've adjusted it so that a regular, safe setup with fully filled plasma canisters will only give about 400,000W which should be enough to power the station, but doesn't quite charge the SMES as fast as possible, to give engineers an incentive to tweak their mixes and setups instead of always going for the boring setup. Time to write up a guide on this engine now.

ChemicalRascal commented:
This is a terrible idea. The SM shard takes a *long* time to get done properly, is *incredibly* fragile, and very few people actually know how to set it up.

Having it as an option is fine. Having it as the *only* option ensures that smart engineers will only ever do solars, and a dumb engineer can fuck over everyone in an instant.

ChemicalRascal commented:
I mean, I get Kor's arguments that the silent, binary failure/success state of the singulo is a problem, but the difficulty involved in setting up the SM makes it such a terrible replacement.

Cyberboss commented:
I'll deal with this when I finally get template engines working
